RepoMan/RepoMan/Program.cs

```diff
@@ -194,46 +172,41 @@ private static ILogger GetLogger()
                 .CreateLogger();
         }
 
+        private static JsonSerializerSettings GetJsonSerializerSettings(IScorerFactory scorerFactory)
```

Comment:
Yeah, honestly I didn't think hard about this -- I just threw it together, because I like being about to actually read JSON during exploration/debugging.